<!-- Parameters for the caros_universalrobot node -->
<launch>
  <!-- Define the parameterisation of the ROS parameters -->
  <arg name="deviceName" default="UR1" />
  <arg name="FTFrame" default="WORLD" />
  <arg name="IP" default="192.168.100.4" />
  <arg name="callbackPort" default="33333" />
  <arg name="callbackIP" default="192.168.100.1" />
  <arg name="WrenchTopic" default="wrenchData" />

  <!-- Set the ROS parameters -->
  <param name="deviceName" type="str" value="$(arg deviceName)" />
  <param name="FTFrame" type="str" value="$(arg FTFrame)"/>
  <param name="IP" type="str" value="$(arg IP)"/>
  <param name="callbackPort" type="str" value="$(arg callbackPort)"/>
  <param name="callbackIP" type="str" value="$(arg callbackIP)"/>
  <param name="WrenchTopic" type="str" value="$(arg WrenchTopic)"/>
</launch>
